### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by v-momoin**
Updated the changelog.md as per the SDK release for November

Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

A few sentences describing the overall goals of the pull request.

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # App Center SDK for React Native Change Log
 
-## Version 1.10.0 (Under development)
+## Version 1.10.0
 
 ### Fixes
```
Comment (guperrot) [inline CHANGELOG.md]: Actually we need to document native SDK bug fixes coming from https://github.com/Microsoft/AppCenter-SDK-Apple/releases/tag/1.11.0 and https://github.com/Microsoft/AppCenter-SDK-Android/releases/tag/1.10.0 that we will integrate in this release.

Features are not applicable though.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,36 @@
 # App Center SDK for React Native Change Log
 
-## Version 1.10.0 (Under development)
+## Version 1.10.0
 
+### Features
+
+AppCenterAnalytics
```
Comment (guperrot) [inline CHANGELOG.md]: Add `####`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,13 +1,36 @@
...
+
+[Android] Add API to specify event persistence priority.
```
Comment (guperrot) [inline CHANGELOG.md]: Add `-` to format list items.
`[Android]` => `**[Android]**`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,13 +1,36 @@
...
+[IOS] Add new trackEvent APIs that take priority (normal or critical) of event logs. Events tracked with critical flag will take precedence over all other logs except crash logs (when AppCenterCrashes is enabled), and only be dropped if storage is full and must make room for newer critical events or crashes logs.
 ### Fixes
```
Comment (guperrot) [inline CHANGELOG.md]: Add blank line before

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,13 +1,36 @@
...
 
 - Fix lint issue on modern projects using latest react-native versions [#451](https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/451).
 - Use latest Firebase version [#365](https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/365).
 - [Android] Fix a bug which prevents ErrorAttachmentLog from being send if fileName is not specified.
 
+AppCenter
```
Comment (guperrot) [inline CHANGELOG.md]: `#### AppCenter`

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,13 +1,36 @@
...
+- [IOS] Fix an issue where concurrent modification of custom properties was not thread safe.
```
Comment (guperrot) [inline CHANGELOG.md]: Add blankline between section header and list of fixes.
Use `**[iOS]**` (bold, lower case i for iOS).

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,13 +1,36 @@
...
+- [IOS] Fix validating and discarding Not a Number (NaN) and infinite double values for custom properties.
+- [IOS] Fix Use standard SQL syntax to avoid affecting users with custom SQLite libraries.
+- [IOS] Fix Get database page size dynamically to support custom values.
+
+AppCenterCrashes
+
+- [Android] Fix Preventing stack overflow crash while reading a huge throwable file.
+- [IOS] Do not force crash macOS application on uncaught exception. If you need this behavior you can set the special flag yourself:
+
+[[NSUserDefaults standardUserDefaults] registerDefaults:@{ @"NSApplicationCrashOnExceptions" : @YES }];
```
Comment (guperrot) [inline CHANGELOG.md]: Code snippet uses ``` (triple back quotes) before and after

File: CHANGELOG.md [hunk 8]
```diff
@@ -24,8 +24,9 @@ AppCenterCrashes
 
 - [Android] Fix Preventing stack overflow crash while reading a huge throwable file.
 - [IOS] Do not force crash macOS application on uncaught exception. If you need this behavior you can set the special flag yourself:
-
-[[NSUserDefaults standardUserDefaults] registerDefaults:@{ @"NSApplicationCrashOnExceptions" : @YES }];
+    ```objc
```
Comment (guperrot) [inline CHANGELOG.md]: add blank line before code snippet

File: CHANGELOG.md (hunk 3 above)
Comment (v-momoin) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 4 above)
Comment (v-momoin) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 5 above)
Comment (v-momoin) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 6 above)
Comment (v-momoin) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 8 above)
Comment (v-momoin) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,12 +1,39 @@
 # App Center SDK for React Native Change Log
 
-## Version 1.10.0 (Under development)
+## Version 1.10.0
+
+### Features
+
+### AppCenterAnalytics
+
+- **[Android]** Add API to specify event persistence priority.
```
Comment (dhei) [inline CHANGELOG.md]: The two "###AppCenterAnalytics" features you mentioned are only for native Android or native iOS, not for RN. Actually there is no new RN features in this release. Only bug-fixes.

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,12 +1,29 @@
...
+    [[NSUserDefaults standardUserDefaults] registerDefaults:@{ @"NSApplicationCrashOnExceptions" : @YES }];
+    ```
+
+### AppCenterPush
 
-- Fix lint issue on modern projects using latest react-native versions [#451](https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/451).
-- Use latest Firebase version [#365](https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/365).
-- [Android] Fix a bug which prevents ErrorAttachmentLog from being send if fileName is not specified.
+- **[Android]** Use latest Firebase version [#365](https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/365).
+- **[iOS]** Fix push:didReceivePushNotification: callback not triggered on notification tapped or received in foreground when a      UNUserNotificationCenterDelegate is set. If you have implemented this delegate please remove any call to the MSPush#didReceiveRemoteNotification: method as it's now handled by the new User Notification Center Delegate Forwarder.
```
[the remark this answers is not in the document]
Comment (dhei) [reply CHANGELOG.md]: addressed my own feedback

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,12 +1,31 @@
...
+- **[iOS]** Fix an issue where concurrent modification of custom properties was not thread safe.
+- **[iOS]** Validating and discarding Not a Number (NaN) and infinite double values for custom properties.
+- **[iOS]** Use standard SQL syntax to avoid affecting users with custom SQLite libraries.
+- **[iOS]** Get database page size dynamically to support custom values.
+
+### AppCenterCrashes
+
+- **[Android]** Fix a bug which prevents attachments from being sent if file name is not specified.
+- **[Android]** Fix Preventing stack overflow crash while reading a huge throwable file.
+- **[iOS]** Do not force crash macOS application on uncaught exception. If you need this behavior you can set the special flag yourself:
```
Comment (dhei) [inline CHANGELOG.md]: This is macOS only while there is no macOS support in RN SDK. No need to mention.

Comment (dhei) [reply CHANGELOG.md]: addressed

**Pull request merged**